app/bundles/CampaignBundle/Form/Type/EventType.php

```diff
@@ -297,9 +303,9 @@ public function configureOptions(OptionsResolver $resolver)
     /**
      * @return string
      */
-    public function getName()
+    public function getBlockPrefix()
     {
-        return 'campaignevent';
+        return self::class;
```

Comment:
I don't see this alias being used in a JS file so the whole method can be removed. However, the alias must be replaced with FQCN in other places. Like `EventController` where the form is being created.